### PR TITLE
Fix sentiment dereference for latest model changes

### PR DESCRIPTION
**Pull request by c-w**
Made necessary by changes in this pull request
https://github.com/CatalystCode/project-fortis-pipeline/pull/118

### CODE REVIEW

File: src/resolvers-cassandra/Tiles/queries.js [hunk 1]
```diff
@@ -9,12 +9,12 @@ const { trackEvent } = require('../../clients/appinsights/AppInsightsClient');
...
 
 /**
- * @param {{tilex: number, tiley: number, tilez: number, avgsentiment: number, mentioncount: number}} rows
+ * @param {{tilex: number, tiley: number, tilez: number, avgsentimentnumerator: number, mentioncount: number}} rows
  */
 function computedtileToTile(row) {
   const coordinates = [geotile.longitudeFromColumn(row.tiley, row.tilez), geotile.latitudeFromRow(row.tilex, row.tilez)];
   const mentionCount = row.mentioncount;
-  const neg_sentiment = row.avgsentiment;
+  const neg_sentiment = row.avgsentimentnumerator / row.mentioncount;
```
Comment (Smarker) [inline src/resolvers-cassandra/Tiles/queries.js]: I'd guard against row.mentioncount = 0

Comment (c-w) [reply src/resolvers-cassandra/Tiles/queries.js]: Division by zero is safe(-ish) in Javascript (just gives you infinity), but sure, let's make it explicit: 62c0ee2

Comment (Smarker) [reply src/resolvers-cassandra/Tiles/queries.js]: Oh interesting I didn't know that.

**Review by Smarker — APPROVED** [review]
LGTM

**Pull request merged**